From ef3030072f0eb23c670b5c56d60ccc65f8b70ee2 Mon Sep 17 00:00:00 2001
From: Zefa Chen <zefa.chen@rock-chips.com>
Date: Mon, 13 Apr 2020 20:17:09 +0800
Subject: [PATCH] ARM: dts: modify camera config for rv1126

ARM: dts: rv1126-evb-ddr3-v10:add support os04a10,
          change module name to match iq files
ARM: dts: rv1126-ipc2-ddr3-v10:
          change module name to match iq files
ARM: dts: rv1126-sphericalipc-ddr3-v10:
          change module name to match iq files

Change-Id: Icd30fd167c2937e466398add575f01cb204b0b05
Signed-off-by: Zefa Chen <zefa.chen@rock-chips.com>
---
 arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts          | 11 ++++-------
 arch/arm/boot/dts/rv1126-ipc2-ddr3-v10.dts         |  6 ++----
 arch/arm/boot/dts/rv1126-sphericalipc-ddr3-v10.dts |  6 ++----
 3 files changed, 8 insertions(+), 15 deletions(-)

diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts
index c5b8588c54ab..8cf06f1b80eb 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts
@@ -20,8 +20,8 @@
 	status = "okay";
 	clock-frequency = <400000>;
 
-	ov4689: ov4689@36 {
-		compatible = "ovti,ov4689";
+	os04a10: os04a10@36 {
+		compatible = "ovti,os04a10";
 		reg = <0x36>;
 		clocks = <&cru CLK_MIPICSI_OUT>;
 		clock-names = "xvclk";
@@ -34,17 +34,14 @@
 		pwdn-gpios = <&gpio1 RK_PD4 GPIO_ACTIVE_HIGH>;
 		rockchip,camera-module-index = <1>;
 		rockchip,camera-module-facing = "front";
-		rockchip,camera-module-name = "JSD3425-C1";
-		rockchip,camera-module-lens-name = "JSD3425-C1";
-		/* NO_HDR:0 HDR_X2:5 HDR_X3:6 */
-		rockchip,camera-hdr-mode = <0>;
+		rockchip,camera-module-name = "CMK-OT1607-FV1";
+		rockchip,camera-module-lens-name = "M12-4IR-4MP-F16";
 		port {
 			ucam_out0: endpoint {
 				remote-endpoint = <&mipi_in_ucam0>;
 				data-lanes = <1 2 3 4>;
 			};
 		};
-
 	};
 };
 
diff --git a/arch/arm/boot/dts/rv1126-ipc2-ddr3-v10.dts b/arch/arm/boot/dts/rv1126-ipc2-ddr3-v10.dts
index b2c7d057bb6d..12649e64c169 100644
--- a/arch/arm/boot/dts/rv1126-ipc2-ddr3-v10.dts
+++ b/arch/arm/boot/dts/rv1126-ipc2-ddr3-v10.dts
@@ -375,10 +375,8 @@
 		pwdn-gpios = <&gpio1 RK_PD4 GPIO_ACTIVE_HIGH>;
 		rockchip,camera-module-index = <1>;
 		rockchip,camera-module-facing = "front";
-		rockchip,camera-module-name = "JSD3425-C1";
-		rockchip,camera-module-lens-name = "JSD3425-C1";
-		/* NO_HDR:0 HDR_X2:5 HDR_X3:6 */
-		rockchip,camera-hdr-mode = <0>;
+		rockchip,camera-module-name = "CMK-OT1607-FV1";
+		rockchip,camera-module-lens-name = "M12-4IR-4MP-F16";
 		port {
 			ucam_out0: endpoint {
 				remote-endpoint = <&mipi_in_ucam0>;
diff --git a/arch/arm/boot/dts/rv1126-sphericalipc-ddr3-v10.dts b/arch/arm/boot/dts/rv1126-sphericalipc-ddr3-v10.dts
index 6e3ad21dc4af..e19d138b916c 100644
--- a/arch/arm/boot/dts/rv1126-sphericalipc-ddr3-v10.dts
+++ b/arch/arm/boot/dts/rv1126-sphericalipc-ddr3-v10.dts
@@ -383,10 +383,8 @@
 		pwdn-gpios = <&gpio1 RK_PD4 GPIO_ACTIVE_HIGH>;
 		rockchip,camera-module-index = <1>;
 		rockchip,camera-module-facing = "front";
-		rockchip,camera-module-name = "default";
-		rockchip,camera-module-lens-name = "default";
-		/* NO_HDR:0 HDR_X2:5 HDR_X3:6 */
-		rockchip,camera-hdr-mode = <5>;
+		rockchip,camera-module-name = "JSD3425-C1";
+		rockchip,camera-module-lens-name = "4IR";
 		port {
 			ucam_out0: endpoint {
 				remote-endpoint = <&mipi_in_ucam0>;
-- 
2.35.3

